woocommerce/woocommerce
# Updating product AI features endpoints

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

Switching the product title generation and description generation features over to using our generic text completion endpoint. 

Closes #38895 



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Checkout branch and activate plugin.
2. Navigate to WooCommerce -> Add new
3. Click on title field, and generate suggestions using box that appears below. This behavior should function as it has previously.
4. Click the "Write with AI" button above the description field. This should stream to the contents of the field. It's behavior is also the same as it was previously.
